analyzers/tests/SonarAnalyzer.Net8.UnitTest/SonarAnalyzer.Net8.UnitTest.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <!-- `-windows` is required in order to be able to reference SonarAnalyzer.UnitTests -->
+        <TargetFramework>net8.0-windows</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="altcover" Version="8.6.68" />
+      <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.6.3" />
+      <PackageReference Include="MSTest.TestAdapter" Version="3.1.1" />
+      <PackageReference Include="MSTest.TestFramework" Version="3.1.1" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <Compile Remove="TestCases\**\*" />
+      <None Include="TestCases\**\*">
+        <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      </None>
+    </ItemGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\SonarAnalyzer.UnitTest\SonarAnalyzer.UnitTest.csproj" />
```

Comment:
I don't think we should make this a permanent solution. The dependency of one UT project on another is highly unusual. https://github.com/SonarSource/sonar-dotnet/issues/8237 addresses this, but I don't understand the rationale behind the separation.

Reply:
@martin-strecker-sonarsource did you read the readme file? I don't understand your review. I've clearly stated that this is a temporary change.
> A temporary decision that will allow us to add support on time for .net 8.

Reply:
The validation API is "main" code and should be seen as such. We should analyze it as production code, test it, and compute coverage for it.

It will also provide a clear separation (tests vs testing framework) and better flexibility allowing us to reorganize the tests and work around limitations as we saw in the `MSBuildWorkspace`.